src/app/(auth)/reset-password/page.tsx

```diff
@@ -22,29 +24,39 @@ export default function ResetPassword() {
   }, []);
 
   const resetPassword = async () => {
+    if (newPassword !== newPassword2) {
+      setErrorMessage('Passwords do not match.');
+      return;
+    }
     const { error } = await supabase.auth.updateUser({ password: newPassword });
     if (error) {
-      throw new Error(
-        `An error occurred trying to reset password: ${error.message}`,
-      );
-    }
-    const { error: signOutError } = await supabase.auth.signOut();
-    if (signOutError) {
-      throw new Error(
-        `An error occurred trying to sign out: ${signOutError.message}`,
-      );
+      setErrorMessage(error.message);
+      // throw new Error(
+      //   `An error occurred trying to reset password: ${error.message}`,
+      // );
+    } else {
+      const { error: signOutError } = await supabase.auth.signOut();
+      if (signOutError) {
+        throw new Error(
+          `An error occurred trying to sign out: ${signOutError.message}`,
+        );
+      }
```

Comment:
oh VERY true, that should definitely be included in this sprint 

@davidqing6432 thoughts? all i can think of is a toast notification

Reply:
yes definitely! i wasn't really sure what to do here, since i don't think a sign out error should occur. @davidqing6432 @pragyakallanagoudar thoughts on error handling?